Added includeArchived param to system log export (#1812)
FusionAuth/fusionauth-site: site/docs/v1/tech/apis/_system-logs-request-body.adoc

+
When this parameter is omitted a default format of `M/d/yyyy hh:mm:ss a z` will be used. See the https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html[DateTimeFormatter patterns] for additional examples.

> [field]#includeArchived# [type]#[Boolean]# [optional]#Optional# [default]#defaults to `false`#::

Reviewer: On request bodies, when there is something new, we include the `[since]` field. For example: `[since]#Available since 1.30.0#` if it was added in version 1.30.0`. 

You can see other examples here: https://raw.githubusercontent.com/FusionAuth/fusionauth-site/master/site/docs/v1/tech/apis/_user-request-body.adoc

Whether to include rotated, compressed logs in addition to the set of active logs in the export result. The default behavior is to only export the set of logs actively being written.
+

> When this parameter is included with a value of `true`, the `lastNBytes` parameter is ignored and all logs are exported with all log data.

Reviewer: You might add a note to `lastNBytes` stating that it is ignored when `includeArchived` is `true`.

> When this parameter is included with a value of `true`, the `lastNBytes` parameter is ignored and all logs are exported with all log data.

Reviewer: ```suggestion
When this parameter is `true`, all available logs are exported. Additionally, if provided, `lastNBytes` parameter is ignored.
```

don't bury the lede :)
